projects/ccf-rui/src/app/modules/content/stage-nav/stage-nav.component.scss

```diff
@@ -22,7 +22,3 @@
     }
   }
 }
-
-.highlighted {
```

Comment:
Why are you changing the stage-nav? It is not part of the components you are working on!

Reply:
I was working on the fix for the slider color bug and decided to add that to the PR. The only thing I changed was the styling for the view slider.

Reply:
It is not good practice to add unrelated stuff to the pr as I'm currently working on the stage it may cause conflicts!

Reply:
Okay sorry, I won't do that again.

Reply:
Also if it is a bug it should have its own pr for fixing it! And please link to the issue such as the bug using standard #[issue number] tags

Reply:
Well this should be reverted in that case! You can easily revert commits in the  git desktop app by looking at the history tab and right clicking and selecting revert commit on the specific commit.